Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
-Red Hat JBoss Data Grid Online Services
-=======================================
+= Red Hat JBoss Data Grid Online Services
 
 Red Hat JBoss Data Grid Online Services consists of a multiple services:
 
-- Caching Service - the simplest, default cache
-- Shared Memory Service - a persistent cache
+- Caching Service - In this service, JBoss Data Grid is configured as an in-memory caching service without persistence. Data in the cache will be lost if the service is stopped or restarted, and therefore, it should not be used as a permanent store for data. It can be used to improve your application’s response time, by keeping frequently and recently accessed data in a high speed in-memory store.
+- Shared Memory Service - In this image, JBoss Data Grid is configured as a shared memory service with fault-tolerance. That means any data stored by the service will be preserved across restarts. Fault-tolerance is achieved by replicating data across multiple nodes. The data is copied to a specific number of nodes in the cluster using the `owners` setting. This ensures that, in the case of failure, the grid will not lose data as long as the number of failed nodes is less than the number of copies (owners). This also works when nodes are shutdown or restarted manually.
 
 The services are based on https://www.redhat.com/en/technologies/jboss-middleware/data-grid[JBoss Data Grid] technology and they expose various endpoints and configurations depending on the use case.
 
-What's inside?
---------------
+== Getting started
 
-This repository has the following structure:
+Before you begin working with JDG Online Services, you will need a working OpenShift cluster as well as `oc` client in your `$PATH`. See https://openshift.io/[OpenShift documentation] how to achieve this.
```

Comment:
For the first sentence I think you need an 'an' in front  of `oc`. Like this: "...you will need a working OpensShift cluster as well as an `oc` client..."

For the last sentence I would say "See the OpenShift documentation for instructions on how to achieve this.

FWIW, I don't see any documentation at openshift.io. Would this be a better link: https://docs.openshift.com/index.html ?

Or perhaps more direct: https://docs.openshift.com/online/welcome/index.html ?

Reply:
Done